Directory.Build.props

```diff
@@ -119,6 +119,10 @@
 
       See Documentation/coding-guidelines/project-guidelines.md for more details on our configurations.
     -->
+    <!-- UGLY hack to supply netfx debugging from VS. -->
+    <TargetGroup Condition="$(Configuration.Contains('netfx'))">netfx</TargetGroup>
```

Comment:
least effort change... only works with netfx. maybe better to check for netfx OR `net\d{2,3}`?

Reply:
Just as a supportive comment, we don't have the infrastructure to run tests for net4* target groups, only for netfx.